dev/com.ibm.ws.microprofile.metrics.monitor/resources/com/ibm/ws/microprofile/metrics/monitor/resources/MonitorMetrics.nlsprops

```diff
@@ -0,0 +1,27 @@
+###############################################################################
+# Copyright (c) 2018 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#CMVCPATHNAME N/A
+#COMPONENTPREFIX CWPMI
+#COMPONENTNAMEFOR CWPMI WebSphere Performance Monitoring MicroProfile Metrics API
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#ISMESSAGEFILE true
+# #########################################################################
+#-----------------------------------------------------------------------------------------------------------------------------
+# Messages used by MonitorMetrics Impl -  range is 2000-2199
+#-----------------------------------------------------------------------------------------------------------------------------
+FEATURE_REGISTERED=CWPMI2001I: Monitoring metrics can be retrieved through mpMetrics.
+FEATURE_REGISTERED.explanation=Feature monitor-1.0 and mpMetrics-1.1 are enabled and monitoring metrics can be retrieved through mpMetrics.
+FEATURE_REGISTERED.useraction=No user action is required.
+
+FEATURE_UNREGISTERED=CWPMI2002I: All monitoring metrics were unregistered from mpMetrics.
+FEATURE_UNREGISTERED.explanation=Either monitor-1.0, mpMetrics-1.1 or both features were removed from the server and all monitoring metrics were unregistered from mpMetrics.
```

Comment:
The "Either" seems out of place. 

Use "monitor-1.0, mpMetrics-1.1 or both features" or "The monitor-1.0 feature, the mpMetrics-1.1 feature, or both features".